addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -479,6 +479,18 @@ $o-color-extras-nesting-selector: '&, .o_colored_level &';
     @return $-palette;
 }
 
+.o_we_shape {
+    --ConnectionShape--size: 64px;
+    --ConnectionShapeSm--size: 32px;
+    --ConnectionShapeLg--size: 33%;
+}
+
+.o_we_shape_btn_content .o_we_shape {
+    --ConnectionShape--size: 24px;
```

Comment:
Can we use the same units for all variables?
I'd go with percentage, it makes sense in this scenario IMO (like  --ConnectionShapeLg--size)